gravity-forms/gw-double-confirmation-fields.php

```diff
@@ -72,6 +72,12 @@ function register_confirmation_fields( $form_id, $fields ) {
 	}
 
 	$gfcf_fields[ $form_id ][] = $fields;
+
+	// Get the current $fields location on array
+	$index = count( $gfcf_fields[ $form_id ] ) - 1;
+
+	// store the custom validation message
+	$gfcf_fields[ $form_id ][ $index ]['validation_message'] = $validation_message;
```

Comment:
I'm curious if it'd be easier/simpler to merge `validation_message` into `$fields` when it's added to `$gfcf_fields[ $form_id ][]`. 🤔